index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="pt-BR">
```
bota a linguagem pra pt-br primo